docs/getting-started/installation.rst

```diff
@@ -35,14 +35,28 @@ After installation the module ``scipp`` can be imported in Python.
 
 To update or remove ``scipp`` use `conda update <https://docs.conda.io/projects/conda/en/latest/commands/update.html>`_ and `conda remove <https://docs.conda.io/projects/conda/en/latest/commands/remove.html>`_.
 
+If you wish to use Scipp with Mantid you may use the following command to create an environment containing both Scipp and Mantid framework:
```

Comment:
At present, this would only work on linux i think?

Reply:
Yes, I'll add a note of that.